packages/e2e/test/web-extension/extension/background/walletManager.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  StoresFactory,
+  WalletFactory,
+  WalletManagerActivateProps,
+  WalletManagerWorker,
+  exposeApi,
+  walletManagerChannel,
+  walletManagerProperties
+} from '@cardano-sdk/web-extension';
+
+import { AsyncKeyAgent } from '@cardano-sdk/key-management';
+import { SingleAddressWallet, storage } from '@cardano-sdk/wallet';
+import {
+  assetProvider,
+  chainHistoryProvider,
+  networkInfoProvider,
+  rewardsProvider,
+  stakePoolProvider,
+  txSubmitProvider,
+  utxoProvider
+} from './config';
+import { logger, walletName } from '../util';
+import { of } from 'rxjs';
+import { runtime } from 'webextension-polyfill';
+
+const walletFactory: WalletFactory = {
+  create: async (
+    { observableWalletName, cardanoServicesUrl: _cardanoServicesUrl }: WalletManagerActivateProps,
```

Comment:
Probably best to omit `cardanoServicesUrl` until it's being used, and capture follow-up work in a ticket.

Reply:
Or keep it and add a comment. Main goal here is to demonstrate expected usage

Reply:
I have made the comment a bit clearer and the parameter is not destructured anymore.
Done here: [4d319c5](https://github.com/input-output-hk/cardano-js-sdk/pull/502/commits/4d319c5241651b905ef6fb7b77000a855f60eb96)